Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -2390,9 +2409,9 @@ void webkitWebViewBaseForwardNextKeyEvent(WebKitWebViewBase* webkitWebViewBase)
     webkitWebViewBase->priv->shouldForwardNextKeyEvent = TRUE;
 }
 
-void webkitWebViewBaseForwardNextWheelEvent(WebKitWebViewBase* webkitWebViewBase)
+void webkitWebViewBaseForwardNextWheelEvent(WebKitWebViewBase* webkitWebViewBase, GdkEvent* event)
 {
-    webkitWebViewBase->priv->shouldForwardNextWheelEvent = true;
+    webkitWebViewBase->priv->shouldForwardNextWheelEvent.append(event);
```

Comment:
So are you taking ownership here or not? It really doesn't look like it. Notably, there is no gdk_event_ref(). But elsewhere you unref events after removing them from this vector.

Notably, we do not hold or transfer ownership via a raw pointer. Use `GRefPtr<GdkEvent>` to hold ownership or `GRefPtr<GdkEvent>&&` to transfer ownership. `GdkEvent*` is fine when not moving ownership.

Reply:
I don't need to take ownership, but I thought ref/unref might be more clear. Using GRefPtr would have been a bit messy to set up because of the difference between gtk3 and gtk4 (gdk_event_copy/g_event_free vs gdk_event_ref/gdk_event_unref).

Anyways, dropped all the ownership since it isn't needed.